### DISCUSSION TITLE
[FEATURE REQUEST] postgres module BYPASSRLS flag support

### DISCUSSION THREAD

**Discussion by ns-bliu** [saltstack/salt#67630] [Feature Requests]
I'm looking to put yugabyte users into highstate using the postgres module/state. The current module doesn't support '[BYPASSRLS](https://www.postgresql.org/docs/9.5/sql-alterrole.html)' flag. Could we extend the execution module [postgres.user_create](https://github.com/saltstack/salt/blob/1b7611958342ba48a842363e3717f8c67439b2f8/salt/modules/postgres.py#L1397) and state module [postgres_user.present](https://github.com/saltstack/salt/blob/1b7611958342ba48a842363e3717f8c67439b2f8/salt/states/postgres_user.py#L34) to include bypassrls into the parameters?

**Comment by welcome** [comment]
Hi there! Welcome to the Salt Community! Thank you for making your first contribution. We have a lengthy process for issues and PRs. Someone from the Core Team will follow up as soon as possible. In the meantime, here’s some information that may help as you continue your Salt journey.
Please be sure to review our [Code of Conduct](https://github.com/saltstack/salt/blob/master/CODE_OF_CONDUCT.md). Also, check out some of our community resources including:

  - [Community Wiki](https://github.com/saltstack/community/wiki)
  - [Salt’s Contributor Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html)
  - [Join our Community Slack](https://join.slack.com/t/saltstackcommunity/shared_invite/zt-3av8jjyf-oBQ2M0vhXOhJpNpRkPWBvg)
  - [IRC on LiberaChat](https://web.libera.chat/#salt)
  - [Salt Project YouTube channel](https://www.youtube.com/channel/UCpveTIucFx9ljGelW63-BWg)
  - [Salt Project Twitch channel](https://www.twitch.tv/saltprojectoss)

There are lots of ways to get involved in our community. Every month, there are around a dozen opportunities to meet with other contributors and the Salt Core team and collaborate in real time. The best way to keep track is by subscribing to the Salt Community Events Calendar.
If you have additional questions, email us at saltproject@vmware.com. We’re glad you’ve joined our community and look forward to doing awesome things with you!
